app/controllers/claims_controller.rb

```diff
@@ -5,6 +5,8 @@ class ClaimsController < ApplicationController
   before_action :end_expired_sessions
   before_action :update_last_seen_at
 
+  after_action :clear_claim_session, if: proc { params[:slug] == "confirmation" && current_claim&.completed? }
```

Comment:
Worth pulling the proc to a named method? Also wondering if the safe-navigation operator is required. Would the code ever get as far as a `current_claim` that is `nil` with the user on the "confirmation" page?

Reply:
Ah, that's a relic from before we relied on the slug. I'll remove it.